packages/hs-react-ui/src/components/Checkbox/Checkbox.tsx

```diff
@@ -37,6 +39,8 @@ export const Label = styled(LabelElement)`
       display: flex;
       align-items: center;
       cursor: pointer;
+      height: 2em;
+      font-size: 1em;
```

Comment:
@aVileBroker This is the change to solve this ticket: https://headstorm.atlassian.net/browse/FOUNDRYUI-68